crates/vm/levm/src/call_frame.rs

```diff
@@ -118,6 +120,7 @@ impl CallFrame {
         gas_limit: U256,
         gas_used: U256,
         depth: usize,
+        create_op_called: bool,
```

Comment:
Maybe we can change this name to `is_create` if you think it's more appropriate.

Reply:
I didn't call it `is_create` because this flag is only set when the callframe is created by the `generic_create` function. In the case of a `CREATE` type transaction, we don't set this flag so the name `is_create` would be confusing